src/github.com/couchbase/sync_gateway/rest/config.go

```diff
@@ -494,7 +494,7 @@ func ParseCommandLine() {
 			config.LogFilePath = logFilePath
 		}
 
-		if skipRunModeValidation != nil {
+		if *skipRunModeValidation == true {
```

Comment:
Yeah, everything seems to be working for me:  https://gist.github.com/tleyden/a96bd43bee91e7e160d4

Can you post the exact config, binary, and cmd line args you are using into a gist + output?